AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -605,8 +605,45 @@ - (void)collectionView:(UICollectionView *)collectionView didEndDisplayingCell:(
   cellNode.scrollView = nil;
 }
 
-#pragma mark -
-#pragma mark Scroll Direction.
+
+- (void)scrollViewDidScroll:(UIScrollView *)scrollView
+{
+  // If a scroll happenes the current range mode needs to go to full
```

Comment:
Okay, that sounds great. Just to confirm, there're no logic changes here, Correct? No need to reply if that is true and I'm glad that the method order now matches.